### ISSUE TITLE
Update please 

### ISSUE THREAD

**Issue by Jay000000001** [QmelZ/Hackustry#25]
Please update the mod to 140.4 I love the mod so much and I would be grateful if you updated it to the 140.4 version.

**Comment by KnightGabriel** [comment]
@QmelZ Yes please don't just abandon this mod you said you would port to groovy or sth and IK you're still active yet you refuse to update it

**Comment by Tojigomo** [comment]
Pls update
![IMG_2239](https://github.com/user-attachments/assets/be193b4c-cc97-47ca-8dcf-2974f8be428b)
